scripts/x-date-pickers-pro.exports.json

```diff
@@ -383,14 +391,13 @@
   { "name": "unstable_useSingleInputTimeRangeField", "kind": "Variable" },
   { "name": "unstable_useTimeField", "kind": "Variable" },
   { "name": "useClearableField", "kind": "Variable" },
+  { "name": "UseClearableFieldResponse", "kind": "TypeAlias" },
   { "name": "UseClearableFieldSlotProps", "kind": "Interface" },
   { "name": "UseClearableFieldSlots", "kind": "Interface" },
   { "name": "UseDateFieldComponentProps", "kind": "TypeAlias" },
-  { "name": "UseDateFieldDefaultizedProps", "kind": "TypeAlias" },
```

Comment:
Maybe these BCs are also worth noting? 🤔

Reply:
These `UseXXXFieldDefaultizedProps` types were not used in any doc example and I don't think there is a real use case to use them outside of the codebase.
But I can add a list in the migration guide, just I won't be able to give any alternative to people :laughing:

Reply:
Fair point.
Mentioning wouldn't hurt it, but could also increase the scariness level of people wanting to migrate. 🙈 😆 
Maybe we could hide it similarly as we hide the adapter BCs? 🤔

Reply:
Fixed in https://github.com/mui/mui-x/pull/12036

Reply:
I just tried doing it and given the short size of the list, the expandable approach did not look great. 
I just added a basic list of the types :+1: